.circleci/config.yml

```diff
@@ -11,8 +11,7 @@ steps:
           libharfbuzz-dev libfribidi-dev libxcb1-dev
         virtualenv --python=python3 .venv
         source .venv/bin/activate
-        pip3 install -r requirements.txt
-        pip3 install -r dev-requirements.txt
+        pip3 install Fabric3==1.14.post1
```

Comment:
Is this the only package needed? 😮 The removed ones seem important

Reply:
Yes, because we don't do really do much else than run Fabric here. I am not really sure why all the other packages were included originally. I also tested this config on a different branch and it ran the staging deployment successfully